configs/det/psenet/README.md

```diff
@@ -43,7 +44,7 @@ The overall architecture of PSENet is presented in Figure 1. It consists of mult
 #### Notes：
 - Context：Training context denoted as {device}x{pieces}-{MS version}{MS mode}, where mindspore mode can be G - graph mode or F - pynative mode with ms function. For example, D910x8-G is for training on 8 pieces of Ascend 910 NPU using graph mode.
 - The training time of PSENet is highly affected by data processing and varies on different machines.
-- On the ICDAR2015 dataset, the input_shape for exported MindIR in the link is `(1,3,1472,2624)`.
+- On the ICDAR2015 dataset, the input_shape for exported MindIR (ResNet152 backbone) in the link is `(1,3,1472,2624)`, and input_shape for ResNet50 backbone in the link is `(1,3,736,1280)`.
```

Comment:
The `input_shape`s to the exported MindIR models trained on ICDAR105 are `(1,3,1472,2624)` for ResNet-152 backbone and `(1,3,736,1280)` for ResNet-50 backbone.